frontend/src/campaign/speech.js

```diff
@@ -179,10 +218,7 @@ export class Speech extends React.Component {
     };
 
     priorityPoint() {
```

Comment:
Since this is just returning a value from the state dict, can we remove the function call and just put the value directly where it's being called?